packages/desktopjs-electron/src/electron.ts

```diff
@@ -45,7 +45,7 @@ export class ElectronContainerWindow extends ContainerWindow {
     }
 
     private get isRemote(): boolean {
-        return ((<any> this.container.internalIpc).send);
+        return !!this.container.internalIpc.send;
```

Comment:
why the double not, forcing bool out of undefined?

Reply:
yes, I just feel more comfortable returning a primitive value than a function. The name "isRemote" should be enough to tell people that it's a boolean and you should use this only to test for true/false, but if someone stores the value, we might leak a function which could hold references to other things in its closure.